src/illuminatio/test_orchestrator.py

```diff
@@ -157,31 +190,31 @@ def _find_or_create_cluster_resources_for_cases(self, cases_dict, api: k8s.clien
         port_mappings = {}
         for from_host_string, target_dict in cases_dict.items():
             from_host = Host.from_identifier(from_host_string)
-            logger.debug("Searching pod for host " + str(from_host))
-            if not (isinstance(from_host, ClusterHost) or isinstance(from_host, GenericClusterHost)):
+            self.logger.debug("Searching pod for host " + str(from_host))
+            if not isinstance(from_host, (ClusterHost, GenericClusterHost)):
                 raise ValueError("Only ClusterHost and GenericClusterHost fromHosts are supported by this Orchestrator")
             namespaces_for_host = self._find_or_create_namespace_for_host(from_host, api)
             from_host = ClusterHost(namespaces_for_host[0].metadata.name, from_host.pod_labels)
-            logger.debug("Updated fromHost with found namespace: " + str(from_host))
+            self.logger.debug("Updated fromHost with found namespace: " + str(from_host))
             pods_for_host = [pod for pod in self._current_pods if from_host.matches(pod)]
             # create pod if none for fromHost is in cluster (and add it to podsForHost)
             if not pods_for_host:
-                logger.debug("Creating dummy pod for host " + str(from_host))
+                self.logger.debug("Creating dummy pod for host " + str(from_host))
                 additional_labels = {ROLE_LABEL: "from_host_dummy", CLEANUP_LABEL: CLEANUP_ALWAYS}
-                container = k8s.client.V1Container(image="nginx:stable", name="dummy")
+                container = k8s.client.V1Container(image=self.docker_image["target"], name="dummy")
```

Comment:
Same like above

Reply:
We also probably should generate a name instead of using `dummy` there could be conflicts (just add a todo).